src/plugins/auto_batch/src/compiled_model.hpp

```diff
@@ -17,64 +18,60 @@ namespace autobatch_plugin {
 
 class AsyncInferRequest;
 
-class CompiledModel : public InferenceEngine::ExecutableNetworkThreadSafeDefault {
+class CompiledModel : public ov::ICompiledModel {
 public:
-    using Ptr = std::shared_ptr<CompiledModel>;
     struct WorkerInferRequest {
-        using Ptr = std::shared_ptr<WorkerInferRequest>;
-        InferenceEngine::SoIInferRequestInternal _inferRequestBatched;
+        ov::SoPtr<ov::IAsyncInferRequest> _inferRequestBatched;
         int _batchSize;
-        InferenceEngine::ThreadSafeQueueWithSize<std::pair<AsyncInferRequest*, InferenceEngine::Task>> _tasks;
-        std::vector<InferenceEngine::Task> _completionTasks;
+        InferenceEngine::ThreadSafeQueueWithSize<
```

Comment:
let's use new API

Reply:
May I know which new API for this `InferenceEngine::ThreadSafeQueueWithSize`? @ilya-lavrenov

Reply:
`src/inference/dev_api/openvino/runtime/threading/thread_safe_containers.hpp` `ov::threading::ThreadSafeQueueWithSize`